bin/get-firefox

```diff
@@ -1,15 +1,16 @@
 #!/usr/bin/env node
 /* eslint-disable require-atomic-updates */
 "use strict";
-
-const index = require('..'),
-    meow = require("meow"),
-    streamToPromise = require("stream-to-promise"),
-    chalk = require("chalk"),
-    Listr = require("listr"),
-    isCI = require("is-ci"),
-    cli = meow({
-        help: `
+const _meow = import('meow');
```

Comment:
what's the advantage of using a dynamic import here? So this can be a cjs module so require just works? That seems a bit backward.

Reply:
That is actually the problem, meow v10 is not a cjs module so no way that you can required it, as far I can tell you can only dynamically import it.

Reply:
What I meant is you could turn this into a module? Or is there no "mjs binary" mode yet?

Reply:
I could try to do it but this package also depends on others that are also not supporting ESM modules yet so maybe is gonna be a lot of work, and I'm short of time right now, if I find time I'll try to turn it into an esm